doc/users/next_whats_new/inset_axes.rst

```diff
@@ -0,0 +1,7 @@
+'Axes.inset_axes' is no longer experimental
+-------------------------
```

Comment:
```suggestion
'Axes.inset_axes' is no longer experimental
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

(Should match the length of the title and be tildes (this helps when combining the snippets into a single page as part of the release process.

Reply:
Done